### DISCUSSION TITLE
Non-rectangular layouts

### DISCUSSION THREAD

**Discussion by gbishop** [UNC-Project-Open-AAC/OS-DPI#253] [Q&A]
I've been thinking about how to implement "buttons" with arbitrary shapes and positions. We can place rectangles anywhere on the VSD but I'm thinking about something more flexible than that. 
Maybe a circular layout or wedges with frequent letters larger and closer?

I think I see how to do it but before I dive in I'm looking for
1. Interest
2. Suggestions for some layout to try.

Thanks
Gary

**Comment by aetkie** [comment]
Not sure if this is close to what you are thinking, but here is a paper with an example of a VSD with a circle rather than a rectangle as the hotspot. 

[Bhana et al. - 2020 - Supporting Communication and Participation in Shar.pdf](https://github.com/user-attachments/files/16908721/Bhana.et.al.-.2020.-.Supporting.Communication.and.Participation.in.Shar.pdf)
